.github/prompts/folder-structure-blueprint-generator.prompts.md

```
@@ -0,0 +1,406 @@
+  - How to extend existing functionality
+  - Where to place new tests
+  - Configuration modification locations
+  
+- **Dependency Patterns**:
+  - How dependencies flow between folders
+  - Import/reference patterns
+  - Dependency injection registration locations
+
+${INCLUDE_FILE_COUNTS ? 
+"- **Content Statistics**:
+  - Files per directory analysis
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)